### PR TITLE
common scripts added

### PR DESCRIPTION
**Pull request by saurabhuja**

### CODE REVIEW

File: OracleDatabase/RAC/OracleRealApplicationClusters/common/scripts/cmdExec
Comment (Djelibeybi) [inline OracleDatabase/RAC/OracleRealApplicationClusters/common/scripts/cmdExec:4]: Please change all copyright lines for _new_ files to:

`Copyright (c) 2024, Oracle and/or its affiliates.`

File: OracleDatabase/RAC/OracleRealApplicationClusters/common/scripts/cmdExec
Comment (Djelibeybi) [inline OracleDatabase/RAC/OracleRealApplicationClusters/common/scripts/cmdExec:5]: Please change all instances of this URL to use `https` instead of `http`.

File: OracleDatabase/RAC/OracleRealApplicationClusters/common/scripts/main.py
Comment (Djelibeybi) [inline OracleDatabase/RAC/OracleRealApplicationClusters/common/scripts/main.py:1]: It's bad practice to use unversioned Python. Please change all instances to `python3`.

**Review by Djelibeybi — CHANGES_REQUESTED** [review]
The copyright lines need to be fixed and the Python shebang lines need to use versioned Python.

**Comment by saurabhuja** [comment]
Thank you @Djelibeybi

**Review by Djelibeybi — APPROVED** [review]

**Comment by Djelibeybi** [comment]
Still needs to be approved by @psaini79 as code owner before I can merge.

**Review by psaini79 — APPROVED** [review]

**Pull request merged**
